packages/utils/src/byte_array.cairo

```diff
@@ -85,7 +85,7 @@ pub fn felt252_to_byte_array(value: felt252) -> ByteArray {
 pub fn u256_from_byte_array_with_offset(arr: @ByteArray, offset: usize, len: usize) -> u256 {
     let total_bytes = arr.len();
     // Return 0 if offset out of bound or len greater than 32 bytes
-    if offset >= total_bytes || len > 32 {
+    if offset >= total_bytes || len > 33 {
```

Comment:
why?

Reply:
I don't think it's correct way to fix this. Could you give me the code that pass the argument `len` which is greater than 32?

Reply:
yeah sure, dming you on telegram